sdk/src/lib.rs

```diff
@@ -12,6 +12,7 @@ pub mod fee_calculator;
 pub mod hash;
 pub mod inflation;
 pub mod instruction;
+pub mod keypairs;
```

Comment:
You just need to move this below as:
```
#[cfg(not(feature = "program"))]
pub mod keypairs;
```
to pass CI checks

Reply:
Thanks!